<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gaitutils menu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="5" column="5">
       <widget class="QPushButton" name="btnTrialVelocity">
        <property name="text">
         <string>Trial velocity plot</string>
        </property>
       </widget>
      </item>
      <item row="3" column="7">
       <widget class="QPushButton" name="btnAutomark">
        <property name="text">
         <string>Automark events</string>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QPushButton" name="btnAutoprocSession">
        <property name="text">
         <string>Autoprocess session</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QPushButton" name="btnKinCons">
        <property name="text">
         <string>Kinematics consistency</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Plot Nexus trial</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="btnEMGCons">
        <property name="text">
         <string>EMG consistency</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Processing</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="7">
       <widget class="QPushButton" name="btnAutoprocTrial">
        <property name="text">
         <string>Autoprocess single trial</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Consistency plots</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="5">
       <widget class="QPushButton" name="btnTimeDistAverage">
        <property name="text">
         <string>Time-distance average</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="5">
       <widget class="QPushButton" name="btnKinAverage">
        <property name="text">
         <string>Kinematics average</string>
        </property>
       </widget>
      </item>
      <item row="0" column="10">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="5">
       <widget class="QPushButton" name="btnMuscleLenCons">
        <property name="text">
         <string>Muscle length consistency</string>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Reports</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="9">
       <widget class="QPushButton" name="btnCreatePDFs">
        <property name="text">
         <string>Create full PDF report...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="9">
       <widget class="QPushButton" name="btnCreateComparison">
        <property name="text">
         <string>Create comparison PDF report...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="9">
       <widget class="QPushButton" name="btnCreateWebReport">
        <property name="text">
         <string>Create web report...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="7">
       <widget class="QPushButton" name="btnPostprocessSession">
        <property name="text">
         <string>Run postprocessing pipelines</string>
        </property>
       </widget>
      </item>
      <item row="6" column="7">
       <widget class="QPushButton" name="btnCopyVideos">
        <property name="text">
         <string>Copy session videos to desktop</string>
        </property>
       </widget>
      </item>
      <item row="5" column="7">
       <widget class="QPushButton" name="btnConvertSessionVideos">
        <property name="text">
         <string>Convert session videos to web format</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cbNexusTrialLayout"/>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btnPlotNexusTrial">
        <property name="text">
         <string>Plot</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>Plot unnormalized</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Output:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="txtOutput"/>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Active web reports (double click to view in a browser):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="NiceListWidget" name="listActiveReports">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnViewReport">
        <property name="text">
         <string>View report in browser</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDeleteReport">
        <property name="text">
         <string>Delete report</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDeleteAllReports">
        <property name="text">
         <string>Delete all reports</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnQuit">
        <property name="text">
         <string>Quit</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1016</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionOpts"/>
   </widget>
   <widget class="QMenu" name="menuAnalysis">
    <property name="title">
     <string>Analysis</string>
    </property>
    <addaction name="actionTardieu_analysis"/>
   </widget>
   <widget class="QMenu" name="menuProcessing">
    <property name="title">
     <string>Processing</string>
    </property>
    <addaction name="actionAutoprocess_session"/>
    <addaction name="actionAutoprocess_single_trial"/>
    <addaction name="actionAutomark_events"/>
    <addaction name="actionRun_postprocessing_pipelines"/>
    <addaction name="actionConvert_session_videos_to_web_format"/>
    <addaction name="actionCopy_session_videos_to_desktop"/>
   </widget>
   <widget class="QMenu" name="menuReports">
    <property name="title">
     <string>Reports</string>
    </property>
    <addaction name="actionCreate_PDF_report"/>
    <addaction name="actionCreate_comparison_PDF_report"/>
    <addaction name="actionCreate_web_report"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuAnalysis"/>
   <addaction name="menuProcessing"/>
   <addaction name="menuReports"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOptions">
   <property name="text">
    <string>Options...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionOptions_2">
   <property name="text">
    <string>Options...</string>
   </property>
  </action>
  <action name="actionOptions_3">
   <property name="text">
    <string>Options...</string>
   </property>
  </action>
  <action name="actionOptions_4">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionOpts">
   <property name="text">
    <string>Options...</string>
   </property>
  </action>
  <action name="actionTardieu_analysis">
   <property name="text">
    <string>Tardieu analysis...</string>
   </property>
  </action>
  <action name="actionAutoprocess_session">
   <property name="text">
    <string>Autoprocess session</string>
   </property>
  </action>
  <action name="actionAutoprocess_single_trial">
   <property name="text">
    <string>Autoprocess single trial</string>
   </property>
  </action>
  <action name="actionAutomark_events">
   <property name="text">
    <string>Automark events</string>
   </property>
  </action>
  <action name="actionRun_postprocessing_pipelines">
   <property name="text">
    <string>Run postprocessing pipelines</string>
   </property>
  </action>
  <action name="actionConvert_session_videos_to_web_format">
   <property name="text">
    <string>Convert session videos to web format</string>
   </property>
  </action>
  <action name="actionCopy_session_videos_to_desktop">
   <property name="text">
    <string>Copy session videos to desktop</string>
   </property>
  </action>
  <action name="actionCreate_PDF_report">
   <property name="text">
    <string>Create PDF report...</string>
   </property>
  </action>
  <action name="actionCreate_comparison_PDF_report">
   <property name="text">
    <string>Create comparison PDF report...</string>
   </property>
  </action>
  <action name="actionCreate_web_report">
   <property name="text">
    <string>Create web report...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NiceListWidget</class>
   <extends>QListWidget</extends>
   <header>plotter_gui</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
